sdktests/client_side_stream_all.go

```diff
@@ -12,7 +12,6 @@ import (
 
 func doClientSideStreamTests(t *ldtest.T) {
 	t.Run("requests", doClientSideStreamRequestTest)
-	t.Run("updates", doClientSideStreamUpdateTests)
```

Comment:
I removed these update tests from the client and server tests because they don't make sense in the fdv2 work. 

Originally these tests were making sure we handled individual flag / segment version values appropriately when updating the in memory store. That doesn't matter anymore (we strictly go off payload version), so we don't need this.

A subsequent commit will bring back something similar when we start interacting with the data stores. But they won't be written this like I would hope. 

--

This deletion goes until the next comment.